cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -859,6 +864,33 @@ instance ToObject NtN.AcceptConnectionsPolicyTrace where
              , "softLimit" .= show softLimit
              ]
 
+instance ( Mock.SimpleCrypto c
+         , Typeable a)
+ => ToObject (Header (Mock.SimpleBlock c a)) where
+  toObject _verb b =
+    mkObject $
+        [ "kind" .= String "SimpleBlockHeader"
+        , "hash" .= condense (blockHash b)
+        , "prevhash" .= condense (blockPrevHash b)
+        , "slotNo" .= condense (blockSlot b)
+        , "blockNo" .= condense (blockNo b) ]
+
+instance ToObject (Header ByronBlock) where
+  toObject _verb b =
+    mkObject $
+        [ "kind" .= String "ByronBlock"
+        , "hash" .= condense (blockHash b)
+        , "prevhash" .= condense (blockPrevHash b)
+        , "slotNo" .= condense (blockSlot b)
+        , "blockNo" .= condense (blockNo b)
+        ] <>
+        case byronHeaderRaw b of
+          Block.ABOBBoundaryHdr{} -> []
+          Block.ABOBBlockHdr h ->
+            [ "delegate" .= (condense . Dlg.delegateVK . Block.delegationCertificate $
+                             Block.headerSignature h)
```

Comment:
Nitpick: redundant parens

Reply:
@mrBliss, I'm not sure they are redundant, because without them `$` cannot be used due to precedence conflict with `.=`.

Reply:
Alright, you can leave it as is. (I think I would introduce a helper function to retrieve the delegate from the header).

Reply:
@mrBliss, done.